sage-ios/SAGE/SAGE/UnverifiedViewController.swift

```diff
@@ -24,7 +42,7 @@ class UnverifiedViewController: UIViewController {
     }
     
     //
-    // MARK: - Public methods
+    // MARK: - Public methods. Used by SignUpController.
```

Comment:
you can remove the second sentence. MARKs should be very concise (if you look at the top bar right above the code display you can click on this menu which will open a list of all methods and properties inside the file. then you can see how MARKs help organize things)